examples/direct_fidelity_estimation.py

```diff
@@ -173,19 +175,19 @@ class PauliTrace:
 
 
 def _estimate_pauli_traces_clifford(
-    n_qubits: int, clifford_state: cirq.CliffordState, n_measured_operators: Optional[int]
+    n_qubits: int, clifford_tableau: cirq.CliffordTableau, n_measured_operators: Optional[int]
```

Comment:
Do we need to pass in the entire tableau? The implementations seems to only need the stabilizers. Can we just pass that so that this method remains agnostic to the representation used to get the stabilizers?

Reply:
Done.